src/modules/PublicLab.TitleModule.Related.js

```diff
@@ -81,15 +129,28 @@ module.exports = function relatedNodes(module) {
  
       if (module.options.suggestRelated) {
         relatedEl.fadeIn();
+        addedRelatedEl.fadeIn();
+        relatedResultShow = true;
         fetchRelated(show);
       }
  
     });
+
+    $(module.el).find('input').click(function(e) {
  
-    $(module.el).find('input').focusout(function(e) {
+      if (module.options.suggestRelated && relatedResultShow) {
```

Comment:
`module.options.suggestRelated && relatedResultShow` it's better to refactore expressions in variables. It's more readable.